pkg/cmd/docgen/diagrams.go

```diff
@@ -158,7 +160,12 @@ var diagramCmd = func() *cobra.Command {
 			if railroadJar != "" {
 				_, err := os.Stat(railroadJar)
 				if err != nil {
-					log.Fatalf("%s not found\n", railroadJar)
+					if envutil.EnvOrDefaultBool("COCKROACH_REQUIRE_RAILROAD", false) {
+						log.Fatalf("%s not found\n", railroadJar)
+					} else {
+						log.Printf("%s not found, falling back to slower web service (employees can find it on google drive).", railroadJar)
```

Comment:
optional nit: the "it" in the parenthetical has an awkward antecedent

Reply:
Done.